### ISSUE TITLE
Feature request: Adding custom support.

### ISSUE THREAD

**Issue by duhadventure** [arlyon/veining#23]
Ability to add block ids to the config to be able to be veined. Then you don't have to add support for ores that do not work because the players can.

**Comment by arlyon** [comment]
You're right this would be useful. I'm waiting for Forge to stabilize for 1.13 / 1.14 then I'm going to put some more time in. Thanks for the request.

**Comment by darchitectlp** [comment]
I know it's been years... but on the off-chance you could add this feature for 1.12 (some of us aren't leaving this version), that'd be awesome!
